CIS194.cabal

```diff
@@ -28,6 +28,12 @@ library
       Lib
     , Week4.HW4
     , Week1.HW1
+<<<<<<< HEAD
```

Comment:
These <<< and >>> usually signify merge conflict in git that needs to be resolved.
In this case this will mostly consist of deleting the 3 lines that make the cabal file invalid.

It is good practice to run `stack build` / `stack test` to ensure the project builds as a whole / tests are passing.

Reply:
I created this PR by cherry-picking some commits but forgot to pick the commit on the cabal file... Thanks for the reminder.